### PR TITLE
View names will appear in query plan instead of the materialized table

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

**Review by isha97 — APPROVED** [review]

File: bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryUtil.java [hunk 1]
```diff
@@ -594,4 +594,11 @@ static Field adjustField(Field field, @Nullable Field existingField) {
     // no adjustment
     return field;
   }
+
+  public static String prepareQueryForLog(String query, int maxLength) {
+    String noNewLinesQuery = query.replace("\n", "");
+    return noNewLinesQuery.length() > maxLength
+        ? noNewLinesQuery.substring(0, maxLength) + /* ellipsis */ '\u2026'
```
Comment (arunkumarchacko) [inline bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryUtil.java:601]: can consider logging the truncated part length.

File: bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryUtil.java [hunk 2]
```diff
@@ -594,4 +594,11 @@ static Field adjustField(Field field, @Nullable Field existingField) {
     // no adjustment
     return field;
   }
+
+  public static String prepareQueryForLog(String query, int maxLength) {
+    String noNewLinesQuery = query.replace("\n", "");
```
Comment (vishalkarve15) [inline bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryUtil.java:599]: A suggestion is to replace "\n" with "\\n". This makes it more readable if there are comments.

File: bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryUtil.java (hunk 1 above)
Comment (davidrabinowitz) [reply bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryUtil.java:601]: The full query is in the log. We put the first 50 characters, and it is just for the query plan

File: bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryUtil.java (hunk 2 above)
Comment (davidrabinowitz) [reply bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryUtil.java]: Done

**Pull request merged**